### PR TITLE
guide: exp sharing methods

### PR DESCRIPTION
**Pull request by jorgeorpinel**
My proposal to address https://github.com/iterative/dvc.org/pull/4172#pullrequestreview-1241277450.

In summary, this simplifies and reorders the sections of [Sharing Experiments](https://dvc.org/doc/user-guide/experiment-management/sharing-experiments), being explicit about pros and cons.

See it in the review app: https://dvc-org-guide-exps-shar-3ai4hw.herokuapp.com/doc/user-guide/experiment-management/sharing-experiments

---

U: this will be part of the product development (new direction for `dvc exp`) and we'll try to sync it with those plans, per https://github.com/iterative/dvc.org/pull/4172#discussion_r1088976140

### CODE REVIEW

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 1]
```diff
@@ -74,82 +24,134 @@ pulling to/from both Git and DVC remotes in the case of experiments.
...
           ▼                       ▼
   ┌─────────────────┐    ┌────────────────┐
-  │   Code and      │    │                │
-  │   metafiles     │    │   Cached data  │  Local project
+  │    Code and     │    │      Data      │
+  │    metafiles    │    │    (cached)    │  Local project
   └─────────────────┘    └────────────────┘
 ```
 
-[dvc experiments]: /doc/user-guide/experiment-management/experiments-overview
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:82]: BTW this old link was broken it seems, and there's more instances throughout the docs...

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 2]
```diff
@@ -4,64 +4,14 @@ In a regular Git workflow, <abbr>DVC repository</abbr> versions are typically
...
-[persistent](/doc/user-guide/experiment-management/persisting-experiments) Git
-commits and share them like you would [any other commit]. This will share all
-code and metadata associated with the experiment.
-
-```cli
-$ dvc exp branch quare-zips my-branch
-Git branch 'my-branch' has been created from experiment 'quare-zips'.
-To switch to the new branch run:
-        git checkout my-branch
+## Method 1: Less steps
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:9]: Other ideas:

- Less steps, less flexible
- For developers
- Stay within the Experiments UI

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 3]
```diff
@@ -74,82 +24,134 @@ pulling to/from both Git and DVC remotes in the case of experiments.
...
 ```
+
+### Limitations
+
+- `dvc exp push` is designed to work with a limited set of experiments (a
+  specific one or based on a Git commit).
+- DVC experiments are not fetched when cloning a <abbr>DVC repository</abbr> (to
+  avoid cluttering your local repo). You must `dvc exp pull` the ones you want.
+
+## Method 2: More persistent
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:105]: Other ideas:

- More steps, more flexible
- For everyone
- Using a regular Git workflow

Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/sharing-experiments.md:105]: > For everyone (vs. "developers")

Hmmm so maybe this should be the first method after all 😅

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 4]
```diff
@@ -74,82 +24,134 @@ pulling to/from both Git and DVC remotes in the case of experiments.
...
+[configure remote storage]:
+  https://dvc.org/doc/user-guide/data-management/remote-storage#configuration
 
 <admon type="tip">
 
-If you don't need to share <abbr>cached</abbr> data, you can skip `dvc remote`
-configuration by using the `dvc exp push --no-cache` and
-`dvc exp pull --no-cache`.
+Check that you have the necessary remotes with `git remote -v` and (optionally)
+`dvc remote list`.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:44]: p.s. I'd probably remove most of these tips (already documented in other guides) to make this section simpler, esp. if we do leave it on top (as method # 1).

**Comment by jorgeorpinel** [comment]
@dberenbaum sorry not sure who to ping about this one. When/if possible, you can see [the proposal here](https://dvc-org-guide-exps-shar-3ai4hw.herokuapp.com/doc/user-guide/experiment-management/sharing-experiments). Feel free to adopt, change, or discard 🙂

**Comment by dberenbaum** [comment]
Thanks @jorgeorpinel! I think your instinct on this one was right. We will probably make more changes here as we continue to build on top of `exp push/pull`, but for now, I made some minor tweaks and think it's ready to merge.

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
Thanks @jorgeorpinel !

**Comment by jorgeorpinel** [comment]
👍🏼

**Pull request merged**